index.js

```diff
@@ -19,6 +21,43 @@ async function mostRecentTag() {
   return versions[0] || semver.parse('0.0.0')
 }
 
+async function getMostRecentBranchTag() {
+  console.log(`Getting list of tags from branch`)
+  let output = ''
+  let err = ''
+  const options = {}
+  options.listeners = {
+    stdout: (data) => {
+      output += data.toString()
+    },
+    stderr: (data) => {
+      err += data.toString()
+    }
+  };
+  options.cwd = './'
+  await exec.exec(`/usr/bin/git`, ['fetch', '--tags', '--quiet'], options)
+  await exec.exec(`/usr/bin/git`, ['tag', '--no-column', '--merged'], options)
```

Comment:
It'd be nice if this could be done in pure js, and maybe using the github API, but this hack works for now.